DQMOffline/Muon/python/muonAnalyzer_cff.py

```diff
@@ -42,3 +42,16 @@
                                   MediumMuonEfficiencyAnalyzer*
                                   TightMuonEfficiencyAnalyzer*                                
                                   muonPFsequence)
+
+from Configuration.StandardSequences.Eras import eras
+for e in [eras.pA_2016]:
+  e.toModify(diMuonHistos, etaBin = cms.int32(175))
+  e.toModify(diMuonHistos, etaBBin = cms.int32(175))
+  e.toModify(diMuonHistos, etaEBin = cms.int32(175))
```

Comment:
minor, somewhat, but still: you can list the modified parameters comma-separated; also, there is not need to specify the type for existing parameters